clickhouse_db_schema/benchmark_regression_summary_report/schema.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE benchmark.benchmark_regression_report
+(
+    `id` UUID DEFAULT generateUUIDv4(),
+    `report_id` String, -- unique id for the report config
+    `created_at` DateTime64(0, 'UTC') DEFAULT now(),
+    `last_record_ts` DateTime64(0, 'UTC'),
+    `last_record_commit` String,
+    `type` String, -- e.g. 'daily','weekly'
+    `status` String, -- e.g. 'no_regression',"regression",'failure'
+    `regression_count` UInt32 DEFAULT 0,
+    `insufficient_data_count` UInt32 DEFAULT 0,
+    `total_count` UInt32 DEFAULT 0,
+    `report` String DEFAULT '{}'
```

Comment:
Till date, I still haven't been able to answer with confidence that CH supports JSON or not, but I guess we can use `JSONExtract` functions here to get things from the report if needed